src/setup-docker/templates/fabric-docker/chaincode-scripts.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Function to perform chaincode invoke
+chaincodeInvoke() {
+  local channel="$1"
+  local chaincode="$2"
+  local peers="$3"
+  local command="$4"
+  local transient="$5"
+
+  echo "-> Chaincode invoke:"
+  echo "Channel: $channel"
+  echo "Chaincode: $chaincode"
+  echo "Peers: $peers"
+  echo "Command: $command"
+  echo "Transient: $transient"
```

Comment:
I think we should use a consistent convention, like the one we use for instance [here](https://github.com/hyperledger-labs/fablo/blob/main/src/setup-docker/templates/fabric-docker/scripts/chaincode-functions-v2.sh):

```
chaincodePackage() {
  local CLI_NAME=$1
  local PEER_ADDRESS=$2
  local CHAINCODE_NAME=$3
  local CHAINCODE_VERSION=$4
  local CHAINCODE_LABEL="${CHAINCODE_NAME}_$CHAINCODE_VERSION"
  local CHAINCODE_LANG=$5

  echo "Packaging chaincode $CHAINCODE_NAME..."
  inputLog "CHAINCODE_VERSION: $CHAINCODE_VERSION"
  inputLog "CHAINCODE_LANG: $CHAINCODE_LANG"
  inputLog "PEER_ADDRESS: $PEER_ADDRESS"
  inputLog "CLI_NAME: $CLI_NAME"
```

So:
* snake case with capital letters for variable names
* and `inputLog` instead of echo